specutils/analysis/correlation.py

```diff
@@ -84,9 +90,14 @@ def template_correlate(observed_spectrum, template_spectrum, lag_units=_KMS,
         normalization = 1.
 
     # Correlate
-    corr = np.correlate(observed_log_spectrum.flux.value,
-                        (template_log_spectrum.flux.value * normalization),
-                        mode='full')
+    if method == "fft":
```

Comment:
```suggestion
    if method.lower() == "fft":
```

Reply:
Thanks for corrections! Now it's look much better :)